apps/condo/domains/common/hooks/useMultipleFiltersModal.tsx

```diff
@@ -455,22 +470,28 @@ const Modal: React.FC<MultipleFiltersModalProps> = ({
             labelCol={LABEL_COL_PROPS}
             initialValue={get(selectedFiltersTemplate, 'name')}
             required
+            rules={[{ required: true, message: FieldRequiredMessage, whitespace: true }]}
```

Comment:
this array will be recreated on each render and have new address in memory

Reply:
done